script/DeployNFTDiscount.s.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8;
+
+import {EthereumScript, BaseScript} from "./utils/ScriptUtils.sol";
+import {NFTDiscount} from "../src/TieredDiscount/NFTDiscount/NFTDiscount.sol";
+
+
+/// make deploy-ledger contract=script/DeployNFTDiscount.s.sol:DeployEthereum chain=mainnet
```

Comment:
any reason why the default is ledger?

also curious about the choice of using make for deploy scripts. syntax is more verbose compared to shell scripts